codecov/settings_base.py

```diff
@@ -28,6 +28,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.contrib.postgres",
+    "django_extensions",
```

Comment:
Adding this little plugin as it has some useful django manage.py commands to make developing easy such as show_urls and graph_models